tests/returntypes/test_jax_qnode_new.py

```diff
@@ -2190,3 +2190,1160 @@ def circuit(x):
 
         assert isinstance(hess[1], jax.numpy.ndarray)
         assert hess[1].shape == (4, 2, 2)
+
+
+shots = [100, (1, 20, 100), (1, (20, 1), 100), (1, (5, 4), 100)]
```

Comment:
To mix up the definition of the shots, I expect that the last `shots` value would have failed prior to the latest changes.
```suggestion
shots = [100, (1, 20, 100), (1, (20, 1), 100), ((5, 4), 1, 100)]
```